src/app.js

```diff
@@ -0,0 +1,54 @@
+const { INGRESS_HOST, INGRESS_PORT, MODULE_NAME } = require('./config/config.js')
+const express = require('express')
+const app = express()
+const winston = require('winston')
+const expressWinston = require('express-winston')
+const { initializeListener } = require('./utils/websocket')
+// initialization
+app.use(express.urlencoded({ extended: true }))
+app.use(express.json())
+
+// logger
+app.use(
+  expressWinston.logger({
+    transports: [
+      new winston.transports.Console(),
+      /*
+    new winston.transports.File({
+        filename: 'logs/websocket-ingress.log'
+    })
+    */
+    ],
+    format: winston.format.combine(winston.format.colorize(), winston.format.json()),
+    meta: true, // optional: control whether you want to log the meta data about the request (default to true)
+    msg: 'HTTP {{req.method}} {{req.url}}', // optional: customize the default logging message. E.g. "{{res.statusCode}} {{req.method}} {{res.responseTime}}ms {{req.url}}"
+    expressFormat: true, // Use the default Express/morgan request formatting. Enabling this will override any msg if true. Will only output colors with colorize set to true
+    colorize: false, // Color the text and status code, using the Express/morgan color palette (text: gray, status: default green, 3XX cyan, 4XX yellow, 5XX red).
+    ignoreRoute: function (req, res) {
+      return false
+    }, // optional: allows to skip some log messages based on request and/or response
+  })
+)
+// main post listener
+app.post('/', async (req, res) => {
+  initializeListener()
+  res.status(200).send({ message: 'Websocket listener initialized' })
+})
+
+// handle exceptions
+app.use(async (err, req, res, next) => {
+  if (res.headersSent) {
+    return next(err)
+  }
+  const errCode = err.status || 401
```

Comment:
why not test equality?

Reply:
this is assignment. no need to change it.

Reply:
I meant `err.status === 401`

Reply:
please leave it as is, I'm sending the code in err.status if for some reason is not present 401 is taken.

Reply:
oh, that's what it means...